frontend/src/App.tsx

```diff
@@ -1,9 +1,23 @@
 import React from 'react';
-import Typeography from '@mui/material/Typography';
+
+import { Route, Routes } from 'react-router-dom';
+
+import AppBar from './components/AppBar';
+import HomePage from './components/HomePage';
+import TagPage from './components/TagPage';
+import DocumentPage from './components/DocumentPage';
 
 function App() {
   return (
-    <Typeography variant='h3' align='center'>React and MUI are installed and setup</Typeography>
+    <div className="App">
+    <AppBar />
+    <Routes>
+        <Route path="/" element={<HomePage />}/>
+        <Route path="/Tags" element={<TagPage />}/>
+        <Route path="/Documents" element={<DocumentPage />}/>
+
```

Comment:
Super minor, but it's usually worth having a catch-all for a 404 page basically, so something along the lines of 
<Route path="*" element={<404Page />}/>

again, super minor